contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -31,11 +32,14 @@ abstract contract FastLaneAuctionHandlerEvents {
     error RelayNotRepaid(uint256 bidAmount, uint256 actualAmount);          // 0x53dc88d9
     error RelaySimulatedNotRepaid(uint256 bidAmount, uint256 actualAmount); // 0xd47ae88a
 
+    error RelayAuctionInvalidBid();
     error RelayAuctionBidReceivedLate();                                    // 0xb61e767e
     error RelayAuctionSearcherNotWinner(uint256 current, uint256 existing); // 0x5db6f7d9
 
     error RelayCannotBeZero();                                              // 0x3c9cfe50
     error RelayCannotBeSelf();                                              // 0x6a64f641
+
+    error RelayValidatorNotAcceptingRefundBids();
```

Comment:
Pleas comment with the selector ( cast sig "RelayValidatorNotAcceptingRefundBids()" )